src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/Manifests/Sample.json

```diff
@@ -0,0 +1,134 @@
+﻿{
+    "version": 5,
+    "data" : {
+        "foo": {
+            "bar": [ 1,2,3]
+        }
+    },
+    "workloads": {
+        // this is a dev workload that would typically be installed by a developer getting
+        // started with this platform. it's composed of several smaller build workloads
+        // plus a template pack for creating projects
+        "xamarin-android": {
+            "description": "Create, build and run Android apps",
+            "kind": "dev",
+            "packs": [
+                "Xamarin.Android.Templates",
```

Comment:
All nuget package name should be all lower case. Since nuget package id is normalized to all lower case. All valid manifest should not have non-lower case package id to avoid down stream mismatch.